recipes/assemblyutility/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set name = "AssemblyUtility" %}
+{% set version = "20160209" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  git_url: https://github.com/yechengxi/AssemblyUtility.git
```

Comment:
After you have pushed something, the most save thing you can do is `git checkout master`, `git pull upstream master` and `git checkout -b new_branch_name` to create and change to a new branch.